src/constants/MarkdownFiles/posts/2025-07-14-gsoc-25-BishoyWadea-week06.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "GSoC ’25 Week 06 Update by Bishoy Wadea"
+excerpt: "Mid term evaluation reflection"
+category: "DEVELOPER NEWS"
+date: "2025-07-13"
+slug: "gsoc-25-BishoyWadea-week06"
+author: "@/constants/MarkdownFiles/authors/bishoy-wadea.md"
+tags: "gsoc25,sugarlabs,week06,midterm,BishoyWadea"
+image: "assets/Images/GSOC.png"
+---
+
+<!-- markdownlint-disable -->
+
+# Week 05 Progress Report by Bishoy Wadea
+
+**Project:** [Euclid’s Game](https://github.com/Bishoywadea/Euclid-s-Game)  
+**Mentors:** [Ibiam Chihurumnaya](https://github.com/chimosky)  
+**Assisting Mentors:** [Walter Bender](https://github.com/walterbender/)  
+**Reporting Period:** 2025-06-29 - 2025-07-05 
+
+---
+
+## Goals for This Week
+
+- **Goal 1:** Start implementing Euclid’s Game
```

Comment:
I recommend adding a little bit more narration to help guide a reader through.

Remind us what Euclid's Game is, and what your goals are in putting it together.